Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY .git .git
 
 # Set UI version 
 RUN echo "export const VERSION = '$(git describe --tag)';" > "src/shared/version.ts"
+RUN echo "export const BUILD_DATE_ISO8601 = '$(date -d "+1 year" -u +"%Y-%m-%dT%H:%M:%SZ")';" > "src/shared/build-date.ts"

```

Comment:
This name is misleading. Better store the current date and then add one year inside the security handler. That or rename the variable.

Reply:
My bad, I'll fix that now.